homework-1/js/script.js

```diff
@@ -0,0 +1,136 @@
+////////////menu///////////
+const iconMenu = document.querySelector('.menu__icon');
+  if(iconMenu) {
+    const menuBody = document.querySelector('.menu__body');
+    iconMenu.addEventListener("click", function(e){
+      document.body.classList.toggle('lock');
+      iconMenu.classList.toggle('active');
+      menuBody.classList.toggle('active');
+    });
+  }
+
+  window.addEventListener('scroll', () => {
+    let scrollDistance = window.scrollY;
+      document.querySelectorAll('.section').forEach((el, i) => {
+        if (el.offsetTop - document.querySelector('#banner-submenu').clientHeight <= scrollDistance) {
```

Comment:
Please move first part of condition to variable. It will be easier to read

Reply:
fixed a bug

Reply:
done